index.html

```diff
@@ -237,16 +237,13 @@ <h2>Scope</h2>
 						Such new technical features should be incubated by the <a href="https://www.w3.org/community/publishingcg/">Publishing Community Group</a> before being considered by this Working Group.  Also, the effects on deployed systems and checking tools, and the risk of additional cost for adoption, must be carefully analyzed. In case of doubt, such features should not be added to EPUB 3.X.
 					</li>
 					<li>
-						Improvements of the <a href='#a11y'>EPUB Accessibility</a> specification; these may include, but not limited to:
+						Improvements to the <a href='#a11y'>EPUB Accessibility</a> specification, that may include, but not be limited to:
 						<ul>
 							<li>
-								verify that EPUB Accessibility 1.X aligns with the <a href='https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2019.151.01.0070.01.ENG&toc=OJ:L:2019:151:TOC'>EU Directive for accessibility requirements for eBooks</a> (European Accessibility Act, Directive 2019/882 of the European Parliament) to the extent possible; the WG will conduct a review of the <a href='#a11y'>EPUB Accessibility</a> specification to ensure that it aligns with the EU accessible publishing requirements to the extent that these improve accessibility;
+								considering early adoption of EPUB Accessibility developments in Europe, while seeking a stronger international consensus through W3C discussion;
 							</li>
```

Comment:
"considering early adoption of EPUB Accessibility developments in Europe, while seeking a stronger international consensus through W3C discussion;"
I believe that the work of a working group is to develop specifications, but here we are going way beyond into promotion and international consensus. This should not be the responsibility of the working group.
The working group should be responsible for aligning EPUB accessibility with requirements of European accessibility act (which could bring it in radar of EU for considering it for EU accessibility act), while facilitating wider adoption internationally.
The following comment is for W3M and not the charter.
I have always seen EPUB Accessibility as a short term thing. Ultimately Silver should be responsible for addressing requirements of EPUB, WP and other possible publishing formats. This is why I want to avoid very long term planning for EPUB Accessibility. I am aware about the big challenges in front of Silver. EPUB accessibility is the need of the hour, but looking beyond 5 or 6 years, we should think of having considerable publishing requirements addressed by Silver.

Reply:
> I believe that the work of a working group is to develop specifications, but here we are going way beyond into promotion and international consensus. This should not be the responsibility of the working group.

in my experience things are never so clear-cut. It is clear that the main job of a WG is to develop specification, but there are many cases when a WG also has to have a role of some level of promotion, seeking contacts, etc. I think that, in view of the circumstances, this is one of those cases...

@plehegar